temporian/core/evaluation.py

```diff
@@ -51,6 +51,60 @@ def evaluate(
 
     The result is returned in the same format as the `query` argument.
 
+
+    Single input output example:
+        ```python
+        >>> input_evset = tp.event_set(timestamps=[1, 2, 3], features={"f": [0, 4, 10]})
+        >>> input_node = input_evset.node()
+        >>> output_node = tp.moving_sum(input_node, 5)
```

Comment:
I read the "# Equivalents" section as being equivalent to the bloc above. What about moving the "output_evset = tp.evaluate(" above the "# Equivalents"?